core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -161,6 +161,12 @@
     font-size: 1.0em;
   }
 
+  @-moz-document url-prefix() {
+    html, body {
+      padding-bottom: 750px;
```

Comment:
I don't think this is the right approach since it introduces unnecessary space at the page below, even though this does work. A better approach is using the `overflow-y` property.

Reply:
@kevinlee12 , PTAL. I have used `overflow-y: auto`  but there seems to be a problem with firefox as it does not work when the `max-height: 100%` is used. Thanks.

Reply:
@kevinlee12 , I removed the `max-height` attribute but it is not working for me.

**Screenshot**
<img width="1440" alt="screenshot 2019-01-27 at 11 37 48 am" src="https://user-images.githubusercontent.com/35570715/51797369-31bcbe00-2228-11e9-91ef-d39bf479fb8d.png">
 Down in the console you can see I have remove `max-height` attribute.

Reply:
Hmm, that's interesting (since it worked on my machine). In that case, let's go back to limiting the height of the feedback text (make it smaller than what you originally had) so that we can see the button. The user will have to scroll the text a bit more, but I think it's an okay trade-off.

Reply:
@kevinlee12 , Should I do it for Firefox only? since it is working fine on chrome.

Reply:
Hmm, let's not, since we want to give users as much of a uniformed experience as possible.

Reply:
Done, ptal.